### ISSUE TITLE
mcdoc (py) error message on giving full comp name if not in ./ 

### ISSUE THREAD

**Issue by willend** [McStasMcXtrace/McCode#717]
Example:

```bash
mcdoc Sample_nxs 
```
gives the usual search page

whereas 
```bash
mcdoc Sample_nxs.comp
is not a file: ./Sample_nxs.comp
```
(Which should have instead found the matching comp file in $MCSTAS/contrib/Sample_nxs.comp)

**Comment by climbcat** [comment]
Ok, which .comp file has priority, the local (if it exists) or the library one?

**Comment by willend** [comment]
Not much need to think about prioroty:

- In case of two matches show the "usual" overview page like this:
<img width="1303" alt="screenshot 2019-01-02 at 10 00 48" src="https://user-images.githubusercontent.com/8593355/50585460-5ade6e80-0e75-11e9-8bfe-1fd62b1184d4.png">

In case of one match only, directly access corresponding html file, be it locally or system-wide. :-)

**Comment by climbcat** [comment]
Ok. Currently I have implemented the prioritized one: 89e5202e10c8196c4b110de4cc258399656715da. Working on the list solution.

**Comment by willend** [comment]
This indeed works, closing
